tgui/packages/tgui/components/Popper.tsx

```diff
@@ -43,7 +45,9 @@ export class Popper extends Component<PopperProps> {
       // This code is copied from `findDOMNode` in inferno-extras.
       // Because this component is written in TypeScript, we will know
       // immediately if this internal variable is removed.
-      const domNode = findDOMfromVNode(this.$LI, true);
+      //
+      // eslint-disable-next-line react/no-find-dom-node
+      const domNode = findDOMNode(this) as Element;
```

Comment:
Either make this use refs or update the comment to remove the mention of Inferno

Reply:
refs are producing the same error unfortunately